src/sonic-yang-mgmt/yang-models/Sonic Yang Tree

```diff
@@ -52,22 +52,37 @@ module: sonic-acl
 module: sonic-interface
```

Comment:
Is file autogenerated?  If so, should we not keep this as part of the code?

Reply:
This file is generated by Pyang tool. Yes, Cool, we can also generate it as part of build process. Sonic-stretch docker will\should have pyang installed and then while building sonic-yang-models pkg I can run "pyang" tool.